crates/router/src/connector/zen/transformers.rs

```diff
@@ -203,12 +203,21 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for ZenPaymentsRequest {
                 ip,
             },
             custom_ipn_url: item.request.get_webhook_url()?,
-            items: vec![ZenItemObject {
-                name: order_details.product_name,
-                price: order_amount.clone(),
-                quantity: 1,
-                line_amount_total: order_amount,
-            }],
+            items: order_details
+                .iter()
+                .map(|data| ZenItemObject {
+                    name: data.product_name.clone(),
+                    quantity: data.quantity,
+                    price: data.amount.to_string(),
+                    line_amount_total: order_amount.clone(),
+                })
+                .collect(),
+            // items: vec![ZenItemObject {
+            //     name: order_details.product_name,
+            //     price: order_amount.clone(),
+            //     quantity: 1,
+            //     line_amount_total: order_amount,
+            // }],
```

Comment:
If this is not required, then please remove it

Reply:
done